demux/samplesheet.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+import click
+import logging
+
+@click.group()
+def samplesheet():
+    """Samplesheet commands"""
+    pass
+
+@samplesheet.command()
+def validate():
+    """TODO: Docstring for validate.
+    Returns: TODO
+
+    """
+    pass
+
+@samplesheet.command()
+def massage():
+    """TODO: Docstring for massage.
+    Returns: TODO
+
+    """
+    pass
+
+@samplesheet.command()
+def demux():
+    """ Convert NIPT samplesheet to demux'able samplesheet """
+    pass
+
+samplesheet.add_command(validate)
```

Comment:
hmm.. I don't think this is necessary since you are using the `@samplesheet.command()` already!